harpy/commands/diagnose.py

```
@@ -62,8 +73,108 @@ def diagnose(directory):
+
+    if not os.path.exists(CONFIG_FILE):
+        print_error("missing workflow config", f"Target directory [blue]{directory}[/] does not contain the file [bold]workflow/workflow.yaml[/]")
+    if not os.path.exists(PROFILE_FILE):
+        print_error("missing snakemake config", f"Target directory [blue]{directory}[/] does not contain the file [bold]workflow/config.yaml[/]")
+
+    with open(CONFIG_FILE, 'r', encoding="utf-8") as f:
+        harpy_config = yaml.full_load(f)
+    
+    command = harpy_config["snakemake"]["absolute"]
+
+    os.system(command)
```

Ruff (0.14.1)

Starting a process with a shell, possible injection detected

(S605)